packages/server/prisma/schema.prisma

```diff
@@ -36,4 +37,5 @@ model Project {
   deletedAt   DateTime?
   homePage    String?
   redirectUrl String?
+  User        User[]
```

Comment:
Do we want the project to keep a list of users, or is it enough to be able to find out the users associated with a project by querying users on the project ID?

Reply:
It's a prisma thing which is required to have when I added `@relation` at the User table. In the migration SQL file, it does not add anything to the Project table.

Reply:
Ok interesting, thanks for the clarification